jb/child_allowance.py

```diff
@@ -1,17 +1,13 @@
-# Roadmap:
-# Comparison to other potential policies
-# Replicate plotly code from the child allowance for JB
-# Move to second simulation
-
 # Preamble and read data
 import microdf as mdf
 import pandas as pd
 import numpy as np
 import us
 
 # Read in census data and specify columns for use
-raw = pd.read_csv(
-    "https://github.com/UBICenter/child-allowance/blob/master/jb/data/cps_00003.csv.gz",
+person_raw = pd.read_csv(
+    "https://github.com/UBICenter/child-allowance/blob/master/jb/data/cps_00003.csv.gz?raw=true",  # noqa
```

Comment:
Load this locally (should also be able to skip `compression="gzip"` after doing so